core/java/iesi-core/src/main/java/io/metadew/iesi/script/action/http/HttpExecuteRequest.java

```diff
@@ -237,6 +237,9 @@ private ProxyConnection convertProxyName(DataType connectionName) {
         if (connectionName == null) {
             return null;
         } else if (connectionName instanceof Text) {
+            if(((Text) connectionName).getString().isEmpty()){
+                return null;
+            }
```

Comment:
The problem occurs when you provide a Null instance as connectionName.
Have a look at other action types to see how they handle the _Null class